pkg/chartutil/create.go

```diff
@@ -63,7 +63,8 @@ service:
   internalPort: 80
 ingress:
   enabled: false
-  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # Used to create Ingress record.
+  # Should be used with service.type: ClusterIP (or NodePort for certain ingress controllers).
```

Comment:
- [ ] Remove this sentence.  
  Some ingress controllers can work with _Services_ that lack a cluster IP ("headless"), let alone them exposing a node port, so it's best to not say anything here about a particular type of _Service_ here, given that the requirements vary this much.